src/app/rosetta/rosetta-cli-config/mina.ros

```diff
@@ -90,3 +90,64 @@ transfer(10){
     ];
   }
 }
+
+// Workflow used to generate delegation transactions
+transfer_delegation(10){
+  transfer{
+    transfer.network = {"network":"debug", "blockchain":"mina"};
+    currency = {"symbol":"MINA", "decimals":9};
+    sender = find_balance({
+      "minimum_balance":{
+        "value": "20000000000",
+        "currency": {{currency}}
+      }
+    });
+
+    // Set the recipient_amount as some value <= sender.balance-max_fee
+    // creation fee of the account if it did not exist before the transaction 
+    max_fee = "4000000000";
+    min_fee = "2000000000";
+
+    // We limit the amount to genereate multiple transactions with the
+    // same prefunded account
+    available_amount = "6000000000";
+
+    recipient_amount = random_number({"minimum": {{max_fee}}, "maximum": {{available_amount}}});
+    print_message({"recipient_amount":{{recipient_amount}}});
+
+    // Find recipient and construct operations
+    sender_amount = 0 - {{recipient_amount}};
```

Comment:
am curious to know why we subtract from 0

Reply:
Writing `- {{recipient_amount}}` causes an error, so it's apparently incorrect for some reason (although it doesn't cause a syntax error in script parsing, so it must be something else…). As for the generation of the ``recipient_amount`, I believe it's just simpler and more obvious to generate a positive value, because it's only a quirk of our Rosetta implementation that requires this value to be negative (as far as I know).